Demo/src/main/res/layout/fragment_main.xml

```diff
@@ -119,4 +119,19 @@
             android:textSize="12sp" />
 
     </TableRow>
+
+    <TableRow
```

Comment:
Can we put this in the row with card rather than it's own row?

Reply:
fixed